openlibrary/templates/type/doc/edit.html

```diff
@@ -1,6 +1,5 @@
 $def with (page)
 
-$ lang = i18n.get_locale() or 'en'
```

Comment:
Let's revert these 3 HTML files; these are _very old_ templates, which used a different system for i18n, I believe.

Reply:
Done.